MainRunner.java

```diff
@@ -0,0 +1,89 @@
+package oopFour;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class MainRunner {
+    static Scanner reader = new Scanner(System.in);
+    public static void main(String[] args) {
+
+        ArrayList<Shape> myShapes = new ArrayList<>();
+
+        Double radius = 0.00;
+        Double length = 0.00;
+        Double width = 0.00;
+        Double sideA = 0.00;
+        Double sideB = 0.00;
+        Double sideC = 0.00;
+        Double height = 0.00;
+
+        do {

```

Comment:
From this menu, I don't see a possibility for user to enter randomly different shapes. 
For example, If I want to enter into the Circle and then do calculation for Rectangle, I don't see a way to do it in current code ?